build/container/Dockerfile

```diff
@@ -64,9 +58,8 @@ RUN OSXCROSS_COMMIT_HASH=86879571d33ff7252a55d777609742561dba9953 && \
     echo "${OSXCROSS_SDK_HASH}  osxcross/tarballs/${OSXCROSS_SDK}" | sha256sum -c - && \
     UNATTENDED=yes OSX_VERSION_MIN=10.6 osxcross/build.sh && \
     mv osxcross/target /usr/local/osxcross && \
-    rm -rf osxcross
-
-ENV PATH /usr/local/osxcross/bin:$PATH
+    rm -rf osxcross && \
+    rm -rf /usr/local/osxcross/SDK/MacOSX10.11.sdk/usr/share
```

Comment:
nit: should the 10.11 in the path here be set to a var like OSXCROSS_COMMIT_HASH is, making it easier to remember to update it in the future?